torch_geometric/data/hetero_data.py

```diff
@@ -778,6 +779,74 @@ def edge_type_subgraph(self, edge_types: List[EdgeType]) -> Self:
                 del data[node_type]
         return data
 
+    def concat(self, hetero_data: Self) -> Self:
+        r"""Concatenates :obj:`self` with another :obj:`hetero_data` object.
+        All values needs to have matching shapes at non-concat dimensions.
+        """
+        out = copy.copy(self)
+        for store, other_store in zip(out.stores, hetero_data.stores):
+            store.concat(other_store)
+        return out
+
+    def is_sorted_by_time(
```

Comment:
Isn't the base implementation sufficient?

Reply:
Well, base implementation is sufficient for all methods, if we want to **always** apply them to all node/edge types. The API provided here is more flexible.

Reply:
@rusty1s, any more comments here? 😉